Userland/Utilities/headless-browser.cpp

```diff
@@ -390,19 +434,24 @@ static ErrorOr<int> run_tests(HeadlessWebContentView& view, StringView test_root
         case TestResult::Timeout:
             ++timeout_count;
             break;
+        case TestResult::Skipped:
+            VERIFY_NOT_REACHED();
+            break;
         }
     }
 
     if (is_tty)
         outln("\33[2K\rDone!");
 
     outln("==================================================");
-    outln("Pass: {}, Fail: {}, Timeout: {}", pass_count, fail_count, timeout_count);
+    outln("Pass: {}, Fail: {}, Skipped: {}, Timeout: {}", pass_count, fail_count, skipped_count, timeout_count);
     outln("==================================================");
     for (auto& test : tests) {
         if (*test.result == TestResult::Pass)
             continue;
-        outln("{}: {}", *test.result == TestResult::Fail ? "Fail" : "Timeout", test.input_path);
+        outln("{}: {}", *test.result == TestResult::Fail ? "Fail" : *test.result == TestResult::Timeout ? "Timeout"
+                                                                                                        : "Skipped",
```

Comment:
This isn't too bad as is, but it's starting to look like a awkward way to avoid a `switch`, or a `test_result_to_string` method for the enum